lib/types/youtube.js

```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```

Comment:
@iefserge that's just not true. The first example is 16:9 while the second example is 4:3. So youtube.com uses both.